lms/static/sass/course/instructor/_instructor_2.scss

```diff
@@ -892,6 +892,217 @@ section.instructor-dashboard-content-2 {
+          padding: ($baseline/2);
+
+          border: 1px solid $light-gray;
+          color: $lighter-base-font-color;
+          line-height: 1.3em;
+          font-size: .85em;
+        }
+      }
+    }
+  }
+
+  .email-list {
+    @include box-sizing(border-box);
+    background-color: $light-gray;
+    margin: ($baseline/4) 0;
+    padding: ($baseline/4);
+
+    table {
+      width: 100%;
+    }
+
+    thead {
+      background-color: dimgrey;
+    }
+
+    tr {
+      border-bottom: 1px solid $light-gray;
+    }
+
+    .or {
+      background-color: lightslategrey;
+    }
+
+    .and {
+      background-color: darkseagreen;
+    }
+
+    .not {
+      background-color: palevioletred;
+    }
+
+    .problem_specific, .section_specific {
+      display: none;
+    }
+
+    .active {
+      display: table-cell;
+    }
+
+    .addingQuery {
+      text-align: right;
+    }
+
+    #addQuery {
+      @include idashbutton($blue);
+      font-size: 12px;
+      position: relative;
+      right: $baseline;
+    }
+
+    #incompleteMessage {
+      padding-right: 40px;
+      color: mediumvioletred;
+    }
+
+    .remove {
+      color: white
+    }
+
+    .remove, .deleteSaved {
+      cursor: pointer;
+      text-align: center;
+      &:hover, &:focus {
+        color: $alert-color;
+      }
+    }
+
+    .deleteSaved {
+      color: black
+    }
+
+
+    .loadQuery {
+      color: black;
+      cursor: pointer;
+      text-align: center;
+
+      &:hover, &:focus {
+        color: $blue;
+      }
+    }
+
+    .tableCaption {
+      text-align: left;
+      padding: 10px;
+      padding-left: 5px;
+      padding-bottom: 10px;
+      font-weight: bold;
+    }
+
+    thead {
+      color:white;
+    }
+
+    td {
+      @extend %t-copy-sub1;
+      vertical-align: middle;
+      word-wrap: break-word;
+      padding: ($baseline/2) ($baseline/4);
+      border-left: 1px solid $light-gray;
+      border-right: 1px solid $light-gray;
+    }
+
+    #estimated {
+      color: dodgerblue;
+      font-size: 20px;
+      padding-bottom: 10px;
+    }
+
+    .subsection {
+      text-indent: 10px;
+    }
+
+    .working {
+      color: black;
+    }
+
+    .bottom-email {
+      @include box-sizing(border-box);
+      @include border-bottom-radius(3);
+      position: relative;
+      padding: ($baseline/2*2);
+      margin-top: -1px;
+      border: 1px solid $light-gray;
+      background-color: #efefef;
+      box-shadow: inset #bbb 0px 1px 1px 0px;
+    }
+
+    input[type="button"]#savequery {
+      @include idashbutton($red);
+      float: right;
+      right: $baseline;
+      font-size: 15px;
+    }
+
+    input[type="button"]#getcsv {
+      @include idashbutton($blue);
+      float: right;
+      right: $baseline;
+      font-size: 15px;
+    }
+
+    .disabled {
+      button:disabled;
+    }
+
+    #queryTableBody {
+      color: white;
+    }
+
+    #savedQueriesTable {
+      background-color: linen;
```

Comment:
Is this an existing convention in our codebase?

You should generally avoid literal CSS color names, as they are not universally supported and can vary from browser to browser.

We should be using either a hex value or a SASS variable (context-dependent) here and elsewhere.